packages/edit-site/src/store/actions.js

```diff
@@ -91,8 +96,8 @@ export function* removeTemplate( templateId ) {
 		path: `/wp/v2/templates/${ templateId }`,
 		method: 'DELETE',
 	} );
-	const page = yield controls.select( 'core/edit-site', 'getPage' );
-	yield controls.dispatch( 'core/edit-site', 'setPage', page );
+	const page = yield controls.select( editSiteStore, 'getPage' );
+	yield controls.dispatch( editSiteStore, 'setPage', page );
```

Comment:
I think these specific lines are culprit to [this error](https://github.com/WordPress/gutenberg/pull/28695/checks?check_run_id=1823394693#step:6:7420).

#27088 only addresses the `select`/`useSelect`/`withSelect` and `dispatch`/`useDispatch`/`withDispatch` exposed methods. I don't think that `controls` can benefit from the `store` definition... In this case, I would take the same approach as [here](https://github.com/WordPress/gutenberg/commit/fd24f069afd5165f33e96d004b2039e94b5f4ceb#diff-ec3b73f716ad961aaf2ea0d6382631f600e7d6dfc60905aa0e3775c102156fa2R43).

Reply:
Just confirming: 

https://github.com/WordPress/gutenberg/blob/16e98d49ffbd32e231099aa687bdc1bbc933c411/packages/data/src/controls.js#L33-L35

The `controls` module doesn't benefit from the `store` definition changes. So, in this case, just remove the hardcoded string with the exported store name.